src/main/java/teammates/ui/controller/InstructorCourseRemindAction.java

```diff
@@ -92,7 +99,14 @@ public ActionResult execute() throws EntityDoesNotExistException {
             }
 
             statusToUser.add(new StatusMessage(Const.StatusMessages.COURSE_REMINDERS_SENT, StatusMessageColor.SUCCESS));
-            redirectUrl = Const.ActionURIs.INSTRUCTOR_COURSE_DETAILS_PAGE;
+
+            /*see and compare if the request is coming from instructorCourseDetails
+            or instructorStudentList and redirect accordingly */
+            if (compareRefererAndCurrentPage(Const.ActionURIs.INSTRUCTOR_COURSE_DETAILS_PAGE)) {
```

Comment:
I don't think the changes in this else clause is necessary. Only the `INSTRUCTOR_COURSE_DETAILS_PAGE` has a remind all students option.